src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -48,387 +102,45 @@ describe('convertInclude', () => {
-    fetch.mockResponse(() => Promise.resolve(vjMarkup));
-    const input = vjBlock;
-    const expected = {
-      type: 'include',
-      model: {
-        href: '/include/111-222-333-444-555',
-        index: 2,
-        isAmpSupported: true,
-        type: 'vj',
-        html: vjMarkup,
-      },
-    };
-    expect(
-      await convertInclude(input, pageData, null, canonicalPathname),
-    ).toEqual(expected);
-    expect(fetch).toHaveBeenCalledTimes(1);
-    expect(fetch).toHaveBeenCalledWith(
-      'https://foobar.com/includes/include/111-222-333-444-555',
-      {
-        timeout: 3000,
+    expectations.forEach(
+      ({
+        block,
+        pathname,
+        expectation,
+        runAdditionalExpectations = () => {},
+      }) => {
+        it(`should return ${expectation}`, async () => {
+          fetch.mockResponse(() => Promise.resolve(includeMarkup));
+          expect(
+            await convertInclude(block, pageData, null, pathname),
+          ).toMatchSnapshot();
+          runAdditionalExpectations();
+        });
       },
     );
-    expect(loggerMock.error).not.toHaveBeenCalled();
-    expect(loggerMock.info).toHaveBeenCalledTimes(1);
-    expect(loggerMock.info).toHaveBeenCalledWith(INCLUDE_REQUEST_RECEIVED, {
-      url: 'https://foobar.com/includes/include/111-222-333-444-555',
-    });
   });
 
-  it('should convert an include block to an idt2 block with html set to null when fetch returns with status other than 200', async () => {
-    fetch.mockResponse(() => Promise.resolve({ status: 304 }));
-    const input = idt2Block;
-    const expected = {
-      type: 'include',
-      model: {
-        href: '/idt2/111-222-333-444-555',
-        index: 1,
-        isAmpSupported: true,
-        type: 'idt2',
-        imageBlock: {
-          alt: 'image alt text',
-          height: 1864,
-          layout: 'responsive',
-          src: 'https://foobar.com/includes/idt2/111-222-333-444-555/image/816',
-          srcset:
-            'https://foobar.com/includes/idt2/111-222-333-444-555/image/470 470w,https://foobar.com/includes/idt2/111-222-333-444-555/image/816 816w',
-          width: 1632,
-        },
-      },
-    };
-    expect(
-      await convertInclude(input, pageData, null, canonicalPathname),
-    ).toEqual(expected);
-    expect(fetch).toHaveBeenCalledTimes(1);
-    expect(fetch).toHaveBeenCalledWith(
-      'https://foobar.com/includes/idt2/111-222-333-444-555/html',
-      {
-        timeout: 3000,
-      },
-    );
-    expect(loggerMock.info).toHaveBeenCalledTimes(1);
-    expect(loggerMock.info).toHaveBeenCalledWith(INCLUDE_REQUEST_RECEIVED, {
-      url: `https://foobar.com/includes/idt2/111-222-333-444-555/html`,
-    });
-    expect(loggerMock.error).toHaveBeenCalledTimes(1);
-    expect(loggerMock.error).toBeCalledWith(INCLUDE_FETCH_ERROR, {
-      status: 304,
-      url: 'https://foobar.com/includes/idt2/111-222-333-444-555/html',
+  describe('Special conditions', () => {
```

Comment:
```suggestion
  describe('Error conditions', () => {
```
Will these conditions only be present when something goes wrong, if so maybe `'Error conditions'` is better?